src/components/Dropdown/components/Control/Control.jsx

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import { components } from "react-select";
+
+const Control = props => {
+  const { selectProps } = props;
+  const control = <components.Control {...props} />;
+  const controlRef = selectProps?.selectProps?.controlRef;
+  if (controlRef)
+    return (
+      <div className="monday-dropdown_scrollable-wrapper" ref={controlRef}>
```

Comment:
did not use css modules here because class name is only for readability purpose. can change and add empty css module file